src/pages/Index.js

```diff
@@ -65,54 +129,189 @@ modalWithImage.setEventListeners();
 /*                                       */
 /*             UserInfo                  */
 /*                                       */
+api
+  .getUserInfo()
```

Comment:
Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all